csharp/sample/Microsoft.ML.OnnxTraining.MNISTSample/Program.cs

```diff
@@ -0,0 +1,175 @@
+﻿using System;
+using System.Collections.Generic;
```

Comment:
File header is missing

Reply:
Please clarify - which file header?

Reply:
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.


---
In reply to: [544535694](https://github.com/microsoft/onnxruntime/pull/6141#discussion_r544535694) [](ancestors = 544535694)

Reply:
Updated.